### PR TITLE
[11.0][MIG] sale_product_set

### PR DESCRIPTION
**Pull request by simahawk**

### CODE REVIEW

**Comment by simahawk** [comment]
@pedrobaeza I'm not sure about:
1. removing encoding on top of file: is this mandatory? If we keep it we'll have less diff compared to v10
2. should I squash older commits into one? Like "Grab module from v10"

**Comment by pedrobaeza** [comment]
> 1. removing encoding on top of file: is this mandatory? If we keep it we'll have less diff compared to v10

In Python 3, it's not needed, but you can keep it if you want. The diff any way is not too big. I'm always in favor of cleaning the maximum possible.

> 2. should I squash older commits into one? Like "Grab module from v10"

In the same wave, this can be an occasion for "cleaning the house" and amend some commits made by the same author in the past. At least, "OCA Transbot..." commits should be squashed together, because lang diffs are not needed.

**Comment by simahawk** [comment]
ok, tnx

**Comment by simahawk** [comment]
@pedrobaeza squashed "meaningfully" and removed encoding headers.

**Comment by pedrobaeza** [comment]
Great, you're the best and what clean is now the PR! :wink:

**Comment by simahawk** [comment]
@pedrobaeza is that a joke? :thinking: :smile: 
image

**Comment by pedrobaeza** [comment]
hahaha, well, pylint is a bit back from Python 3. See https://github.com/OCA/pylint-odoo/pull/168

But that's not your problem. They are only warnings. The real error is the file with executable permissions. See the error counter for figuring out that utf-8 comments can't be.

File: sale_product_set/README.rst [hunk 1]
```diff
@@ -11,11 +11,11 @@ list is defined by sales manager.
...
 at once in a quotation.
 
-After a *product set* is added to the sale order, each line can be update or
-remove as any other sale order lines.
+After a *product set* is added to the sale order, each line can be updated or
+removed as any other sale order lines.
 
-This is differ to packing products as you don't follow *product set* after it
-was added to the sale order.
+This differs from packing products as you don't follow *product set*
```
Comment (gdgellatly) [inline sale_product_set/README.rst]: This sentence makes no sense, perhaps you meant to delete 'as you don't not follow'

File: sale_product_set/views/product_set.xml [hunk 2]
```diff
@@ -72,8 +72,8 @@
         <field name="view_id" ref="view_product_set_tree"/>
     </record>
 
-    <menuitem id="menu_product_set"
-        parent="sales_team.menu_sales"
+    <menuitem id="menu_product_set_config"
+        parent="sale.menu_sales_config"
```
Comment (gdgellatly) [inline sale_product_set/views/product_set.xml]: Is it Config or Catalog?  Just an opinion but I'd say it is more catalog. - non blocking

**Review by gdgellatly — COMMENTED** [review]
1 sentence in docs, otherwise all good by me.

File: sale_product_set/views/product_set.xml (hunk 2 above)
Comment (simahawk) [reply sale_product_set/views/product_set.xml]: sounds more like config but I have no constrain... ;)

File: sale_product_set/__manifest__.py [hunk 3]
```diff
@@ -0,0 +1,27 @@
+# -*- coding: utf-8 -*-
+# Copyright 2015 Anybox
+# Copyright 2017 Camptocamp
```
Comment (ghost) [inline sale_product_set/__manifest__.py]: 2017 -> 2018

File: sale_product_set/models/product_set.py [hunk 4]
```diff
@@ -0,0 +1,21 @@
+# -*- coding: utf-8 -*-
+# Copyright 2015 Anybox S.A.S
+# Copyright 2016-2017 Camptocamp SA
```
Comment (ghost) [inline sale_product_set/models/product_set.py]: 2017 -> 2018

File: sale_product_set/models/product_set_line.py [hunk 5]
```diff
@@ -0,0 +1,36 @@
+# -*- coding: utf-8 -*-
+# Copyright 2015 Anybox S.A.S
+# Copyright 2016-2017 Camptocamp SA
```
Comment (ghost) [inline sale_product_set/models/product_set_line.py]: 2017 -> 2018

File: sale_product_set/tests/test_product_set.py [hunk 6]
```diff
@@ -0,0 +1,69 @@
+# -*- coding: utf-8 -*-
+# Copyright 2015 Anybox S.A.S
+# Copyright 2016-2017 Camptocamp SA
```
Comment (ghost) [inline sale_product_set/tests/test_product_set.py]: 2017 -> 2018

File: sale_product_set/wizard/product_set_add.py [hunk 7]
```diff
@@ -0,0 +1,49 @@
+# -*- coding: utf-8 -*-
+# Copyright 2015 Anybox S.A.S
+# Copyright 2016-2017 Camptocamp SA
```
Comment (ghost) [inline sale_product_set/wizard/product_set_add.py]: 2017 -> 2018

**Review by ghost — APPROVED** [review]
LGTM!

**Comment by simahawk** [comment]
@pedrobaeza @gdgellatly ping

**Review by gdgellatly — APPROVED** [review]

**Review by TDu — APPROVED** [review]

**Review by leemannd — APPROVED** [review]
LGTM! Code review only.

**Pull request merged**